selfdrive/car/interfaces.py

```diff
@@ -97,6 +97,7 @@ def get_std_params(candidate, fingerprint):
     ret.longitudinalTuning.kpV = [1.]
     ret.longitudinalTuning.kiBP = [0.]
     ret.longitudinalTuning.kiV = [1.]
+    # TODO estimate car specific lag, use .15s for now
```

Comment:
Rav4 TSS2 seems to be:
```
Double the ActuatorDelay lag to .3 so we look into the future just a bit
more, seems a bit more inline with realworld experience.```
https://github.com/krkeegan/openpilot/commit/7000a7147c7be3f5ded6921623a9fcd875c2d8fb#diff-c933b99dbed0471164bc0132e6c5dcb8d32ce3e32051cf1309d60c6474c1c332R242

Reply:
Probably will want to make another PR for this, and back up with data 🙂